modules/storage-create-storage-class.adoc

```diff
@@ -49,3 +51,33 @@ endif::[]
 
 // Undefine {StorageClass} attribute, so that any mistakes are easily spotted
 :!StorageClass:
+
+To create the AWS EFS storage class from the CLI, follow these steps:
```

Comment:
I think this should be a separate procedural module.  Also change steps to step since its just one step.

Reply:
* +1 to a separate modules
* -1 to intro statements in the procedure section at all (That should be covered in the abstract/module intro.)
* +1 to removing xrefs from modules

Reply:
Agree completely with @kalexand-rh